### PR TITLE
Extra 0 check

### PR DESCRIPTION
**Pull request by FrederikRothe**
## Describe your changes
Quick change such that we don't get 'empty graphs' when outcome is 0...

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [x] Have I requested and notified the boys for review of the pull-request?

### CODE REVIEW

**Review by Kobo-coder — COMMENTED** [review]
There are no more empty graphs, and that's nice, however there are few things I'm worried of.

If I delete all previous data and start fresh, of the three aggregations only complexity is shown in a graph while duplication is not. I guess it makes sense that maintainability doesn't have a graph since it is described in letters, however seeing `WARNING:root:(No data to plot created for, 'Maintainability')` is mildly confusing (at least for me). For me, a warning sounds like something is not right, but at the same time we deliberately don't show a graph for maintainability.

Hope it makes sense.

File: figures/line.py [hunk 1]
```diff
@@ -23,7 +23,8 @@ def _get_line(self, data, key):
         )
         p.line(x, y, legend_label="Progress", line_width=2)
         try:
-            p.y_range = Range1d(0, (int(max(y)) * 1.1))
+            if max(y) != "0":
+                p.y_range = Range1d(0, (int(max(y)) * 1.1))
```
Comment (SadCheeto) [inline figures/line.py:27]: should we have an else case here? some defaulted value or something similar

**Review by SadCheeto — COMMENTED** [review]
left a comment to look at

File: figures/line.py (hunk 1 above)
Comment (FrederikRothe) [reply figures/line.py:27]: Shouldn't be necessary, we just want to make sure that we handle it different if it's 0. (Then we wont touch the axis ranges)

Comment (FrederikRothe) [reply figures/line.py:27]: My thought was that the else statement would be excessive and unneeded, but I might not see something

**Review by SadCheeto — APPROVED** [review]

**Pull request merged**